superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm/index.tsx

```diff
@@ -73,6 +73,7 @@ export interface FieldPropTypes {
     onRemoveTableCatalog: (idx: number) => void;
```

Comment:
also we need to check the available endpoint to see if a given engine allows ssh tunneling
https://github.com/apache/superset/blob/master/superset/db_engine_specs/base.py#L196

Reply:
Oh! Ok, yeah, IIRC there a `engine_information` property we are getting in the UI with the `supports_file_upload`, so we might need to include this one there and use it. Thanks!